include/tack/float4.h

```diff
@@ -6,6 +6,293 @@
 #include <x86intrin.h>
 
 namespace tack {
+enum blend_t : int32_t {
+    e_aaaa = 0,
+    e_baaa,
+    e_abaa,
+    e_bbaa,
+    e_aaba,
+    e_baba,
+    e_abba,
+    e_bbba,
+    e_aaab,
+    e_baab,
+    e_abab,
+    e_bbab,
+    e_aabb,
+    e_babb,
+    e_abbb,
+    e_bbbb,
+};
+
+enum swizzle_t : int32_t {
+    e_xxxx = 0,
```

Comment:
I'll make this change.  Question though.. should the other enums move with it too?  They might also be shared?